client/my-sites/plan-features/index.jsx

```diff
@@ -70,14 +71,22 @@ class PlanFeatures extends Component {
 
 PlanFeatures.propTypes = {
 	onUgradeClick: PropTypes.func,
-	plan: React.PropTypes.oneOf( [ PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS ] ).isRequired
+	// either you specify the plan prop or isPlaceholder prop
```

Comment:
Besides testing purposes, I'm not sure we'd pass through an isPlaceholder prop for normal use.